### ISSUE TITLE
Address not saving to Salesforce

### ISSUE THREAD

**Issue by fionawhim** [CityOfBoston/311#599]
Sent text:

api_key=WgJlZUOs%2BX%2FxWEmv1dHZ1rPCovs6Kmz%2FSyGb7Z4WWEY%3D&service_code=ILGLDUMP&first_name=Fin&last_name=Hopkins&email=fin.hopkins%40boston.gov&address_string=1+Washington+Mall%0ABoston%2C+02108&address_id=171349&lat=42.3597386&long=-71.0578075

Got response:

```
[{
	"service_request_id": "17-00006161",
	"status": "open",
	"long": null,
	"lat": null,
	"media_url": null,
	"service_name": "Illegal Dumping",
	"service_code": "ILGLDUMP",
	"description": null,
	"requested_datetime": "2017-08-24T17:52:24.000Z",
	"expected_datetime": "2017-08-25T17:52:24.000Z",
	"updated_datetime": null,
	"address": null,
	"zipcode": null,
	"address_id": null,
	"agency_responsible": null,
	"service_notice": null,
	"status_notes": null,
	"contact": {
		"first_name": "Fin",
		"last_name": "Hopkins",
		"phone": null,
		"email": "fin.hopkins@boston.gov"
	},
	"activities": [{
		"code": "ILGLDUMP-INSP",
		"order": null,
		"description": null,
		"status": "Not Started",
		"completion_date": null
	}],
	"attributes": [{
		"code": "BTD-LICNO",
		"description": "If available, please provide license plate number:",
		"order": 1,
		"values": [{
			"answer": null,
			"answer_value": null
		}]
	}, {
		"code": "SR-ILGLDUMP1",
		"description": "What was illegally dumped?",
		"order": 2,
		"values": []
	}, {
		"code": "CE-INCDNTWITNES",
		"description": "Have you witnessed the incident?",
		"order": 3,
		"values": []
	}]
}]
```

address fields are not filled out.

**Comment by fionawhim** [comment]
Location is not saved if `address_id` is specified. Currently waiting to find out what to put there.

**Comment by fionawhim** [comment]
https://incapsulate.atlassian.net/browse/BOS311API-32

**Comment by fionawhim** [comment]
Fixed with #670
